website/pages/blog/product-updates-4.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "CloudQuery Product Updates #4"
+tag: product
+date: 2023/02/27
+description: >-
+  Monthly updates on CloudQuery product and roadmap.
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Our monthly round-up of CloudQuery product and roadmap updates is here! This month was packed!
+
+## SDK Updates
+
+- Better Migrations that supports safe and force for all destinations [688](https://github.com/cloudquery/plugin-sdk/pull/688)!
+
+## Destinations
+
+### New Destinations
+
+We've added 4 new destinations to CQ!
+
+* [Elasticsearch](/docs/plugins/destinations/elasticsearch/overview)
+* [ClickHouse](/docs/plugins/destinations/clickhouse/overview)
+* [Firehose](/docs/plugins/destinations/firehose/overview)
+* [MySQL](/docs/plugins/destinations/mysql/overview)
+
+## Sources
+
+### New Sources
+
+We've welcomed 4 partners and community plugins!
+
+- [Scaleway](https://www.cloudquery.io/docs/plugins/sources/overview#partners)
+- [Simple Analytics](https://www.cloudquery.io/docs/plugins/sources/overview#partners)
+- [Swetrix](https://www.cloudquery.io/docs/plugins/sources/overview#partners)
+- [SharePoint](https://www.cloudquery.io/docs/plugins/sources/overview#community-plugins)
+
+We've added 5 new official source plugins to CQ both in the infra and analytics space!
+
+* [PostgreSQL CDC](/docs/plugins/sources/postgresql/overview)
+* [Homebrew](/docs/plugins/sources/homebrew/overview)
+* [HubSpot](/docs/plugins/sources/hubspot/overview)
+* [Mixpanel](/docs/plugins/sources/mixpanel/overview)
+* [Plausible Analytics](/docs/plugins/sources/plausible/overview)
+* [Launch Darkly](/docs/plugins/sources/launchdarkly/overview)
+
+Numerous updates to [AWS](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/CHANGELOG.md), [GCP](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gcp/CHANGELOG.md) and [Azure](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/azure/CHANGELOG.md).
```

Comment:
It would be nice if we could summarize the newly added tables here (though we should probably improve our changelogs first, so we can see it from there)

Reply:
Yeah I think maybe changelog first improvement first.